man/org.freedesktop.login1.xml

```diff
@@ -0,0 +1,982 @@
+      readonly t InhibitDelayMaxUSec = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly t UserStopDelayUSec = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly s HandlePowerKey = '...';
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly s HandleSuspendKey = '...';
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly s HandleHibernateKey = '...';
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly s HandleLidSwitch = '...';
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly s HandleLidSwitchExternalPower = '...';
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly s HandleLidSwitchDocked = '...';
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly t HoldoffTimeoutUSec = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly s IdleAction = '...';
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly t IdleActionUSec = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
+      readonly b PreparingForShutdown = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
+      readonly b PreparingForSleep = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
+      readonly (st) ScheduledShutdown = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
+      readonly b Docked = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
+      readonly b LidClosed = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
+      readonly b OnExternalPower = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly b RemoveIPC = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly t RuntimeDirectorySize = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly t InhibitorsMax = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
+      readonly t NCurrentInhibitors = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly t SessionsMax = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
+      readonly t NCurrentSessions = ...;
+  };
+  interface org.freedesktop.DBus.Peer { ... };
+  interface org.freedesktop.DBus.Introspectable { ... };
+  interface org.freedesktop.DBus.Properties { ... };
+};
+    </programlisting>
+
+    <refsect2>
+      <title>Methods</title>
+
+      <para><function>GetSession()</function> may be used to get the session object path for the session with
+      the specified ID. Similarly, <function>GetUser()</function> and <function>GetSeat()</function> get the
+      user and seat objects, respectively. <function>GetSessionByPID()</function> and
```

Comment:
We seem to always use a comma when "respectively" is a the end of a sentence. And when reading this aloud, I would insert a pause before "respectively".

Reply:
My bad, I didn't search thoroughly enough when looking if the comma was needed or not. Seems it can be done both ways just like the oxford comma so I'm fine with keeping the commas.